corehq/motech/dhis2/templates/dhis2/dataset_map_list.html

```diff
@@ -96,4 +96,5 @@ <h3>
     <td data-bind="text: ucr"></td>
     <td>{% trans 'Deleted' %}</td>
   </script>
+
```

Comment:
Pro tip: You can rerun Travis by clicking "Details" ...
![image](https://user-images.githubusercontent.com/708421/119816754-5f47ce00-beed-11eb-926f-aeb681977f11.png)
... opening the failed job(s), and clicking "Restart job" in the top right.